docs/_tutorials/getting_started.md

```diff
@@ -13,3 +13,344 @@ redirect_from:
+
+For brevity, we're going to use bot tokens for this guide.
+
+Navigate to the **OAuth & Permissions** on the left sidebar and scroll down to the **Bot Token Scopes** section. Click **Add an OAuth Scope**.
+
+For now, we'll just add one scope: [`chat:write`](https://api.slack.com/scopes/chat:write). This grants your app the permission to post messages in channels it's a member of.
+
+Scroll up to the top of the OAuth & Permissions page and click **Install App to Workspace**. You'll be led through Slack's OAuth UI, where you should allow your app to be installed to your development workspace.
+
+Once you authorize the installation, you'll land on the **OAuth & Permissions** page and see a **Bot User OAuth Access Token**.
+
+<!--TODO - Update image to match the latest App Directory sidebar (but remove the Workflows beta) -->
+![OAuth Tokens](../assets/bot-token.png "Bot OAuth Token")
+
+> 💡 Treat your token like a password and [keep it safe](https://api.slack.com/docs/oauth-safety). Your app uses it to post and retrieve information from Slack workspaces.
+
+---
+
+### Setting up your local project
+With the initial configuration handled, it's time to set up a new Bolt project. This is where you'll write the code that handles the logic for your app.
+
+If you don’t already have a project, let’s create a new one. Create an empty directory:
+
+```shell
+mkdir first-bolt-app
+cd first-bolt-app
+```
+
+Next, we recommend using a Python virtual environment to manage your project's dependencies. Create and activate a new virtual environment:
+
+<details>
+<summary markdown="0">
+<h4>Setting up a Python virtual environment</h4>
+</summary>
+
+TODO - This is where we explain how to installing a Python version manager, virtual environment, and adding it to gitignore.
+
+---
+</details>
+
+<!-- TODO - Details on installing Python version manager, virtual environment, adding it to the git ignore, etc -->
+<!-- TODO - Do we use python or python3? -->
+<!-- TODO - Should we confirm that the virtual environment is active? -->
+```shell
+python3 -m venv env
+source env/bin/activate
+```
+
+Before we install the Bolt for Python package to your new project, let's save the bot token and signing secret that was generated when you configured your app. These should be stored as environment variables and should *not* be saved in version control.
+
+1. **Copy your Signing Secret from the Basic Information page** and then store it in a new environment variable. The following example works on Linux and MacOS; but [similar commands are available on Windows](https://superuser.com/questions/212150/how-to-set-env-variable-in-windows-cmd-line/212153#212153).
```

Comment:
nit: in these days, s/MacOS/macOS/

Reply:
Good catch! Fixed in bf02931daa4481bbd25181371d34a8185e589d46 and I'll add it to a batch of improvements for the Bolt-JS Getting Started guide.